src/reference/90-Developers-Guide/02-Coding-Guideline/00.md

```diff
@@ -175,13 +179,13 @@ object FooID {
 }
 ```
 
-#### Implicit converter for syntax (enrich-my-library) should be imported
+#### Implicit conversions for syntax (enrich-my-library pattern) should be imported
```

Comment:
What's imported are the converters (functions from `A => B`), not conversions (applications to the functions), I think.